test/helpers/kubectl.go

```diff
@@ -541,11 +559,8 @@ func (kub *Kubectl) CiliumPolicyRevision(pod string) (int, error) {
 // CiliumIsPolicyLoaded returns true if the policy is loaded in the given
 // cilium Pod. it returns false in case that the policy is not in place
 func (kub *Kubectl) CiliumIsPolicyLoaded(pod string, policyCmd string) bool {
-	_, err := kub.ExecPodCmd(KubeSystemNamespace, pod, fmt.Sprintf("cilium policy get %s", policyCmd))
-	if err == nil {
-		return true
-	}
-	return false
+	cmd := kub.CiliumExec(pod, fmt.Sprintf("cilium policy get %s", policyCmd))
```

Comment:
Can you rename this from `cmd` to `res`? It adheres to the convention set throughout the Ginkgo framework.